src/components/dashboard/Dashboard.js

```diff
@@ -256,34 +256,31 @@ const Dashboard = props => {
   }
 
   useEffect(() => {
-    if (appState === 'active') {
+    if (appState === 'active' && Number(entitlement)) {
       animateClaim()
     }
-  }, [appState])
+  }, [appState, entitlement])
 
   const animateClaim = useCallback(async () => {
     const inQueue = await userStorage.userProperties.get('claimQueueAdded')
     if (inQueue && inQueue.status === 'pending') {
```

Comment:
claim button uses useClaimQueue hook and determines claim queue status. we could pass it via callback to the Dashboard instead of direct accessing of the user props. Let the all claim queue logic will contain in the custom hook